packages/browser/__tests__/ClientAuthentication.spec.ts

```diff
@@ -113,19 +110,10 @@ describe("ClientAuthentication", () => {
   describe("fetch", () => {
     it("calls fetch", async () => {
       const clientAuthn = getClientAuthentication();
-      const response = await clientAuthn.fetch(
-        "mySession",
+      await clientAuthn.fetch("https://zombo.com");
```

Comment:
Unfortunately the old site uses Flash :P

```suggestion
      await clientAuthn.fetch("https://html5zombo.com");
```

Reply:
Oh my, I didn't realize that was an actual website. And what a website !

Reply:
Resolved in 01046978d8b6d895ed113c543d8dfc8d940c606f